src/Orleans/Runtime/RequestContext.cs

```diff
@@ -42,6 +41,11 @@ public static class RequestContext
         internal const string ORLEANS_REQUEST_CONTEXT_KEY = "#ORL_RC";
         internal const string PING_APPLICATION_HEADER = "Ping";
 
+#if NETSTANDARD_TODO
+        private static readonly AsyncLocal<Guid> ActivityId = new AsyncLocal<Guid>();
+        private static readonly AsyncLocal<Dictionary<string, object>> CallContextData = new AsyncLocal<Dictionary<string, object>>();
```

Comment:
So you think end user should be able to set ActivityID? If we just want to preserve original logic, we should make it public.